.github/actions/action-connect-ovpn/.github/workflows/check_vpn.yml

```diff
@@ -0,0 +1,29 @@
+name: Check connect vpn
+
+on: pull_request   
+
+jobs:
+  check-vpn:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Open VPN
+        run: sudo apt-get install openvpn
+      - name: Connect VPN
+        uses: ./
+        id: connect_vpn
+        with:
+          PING_URL: '${{ secrets.PING_URL }}'
```

Comment:
Secret

Reply:
We do not have this specific secret in this repo. I will continue to review this workflow.

Reply:
All these issues are not documented and we are in the process of reaching out to Mr. Josiah Siegel and ask for his assistance in figuring out the different environment and secrets we need to setup in this project.